tests/test_simple_voice_chat_config.py

```
@@ -0,0 +1,310 @@
+        
+        # Configure session.receive to yield these events then stop
+        # Use a list for the AsyncIterator helper
+        receive_events = [mock_tool_call_event, mock_final_event]
+        mock_session.receive.return_value = AsyncIterator(receive_events)
+
+        # Start the handler (this will start the _audio_sender_task and the receive loop)
+        start_up_task = asyncio.create_task(handler.start_up())
+
+        # Simulate receiving an audio frame
+        sample_audio_bytes = b'\x01\x02\x03\x04'
+        await handler.receive((16000, np.array([1, 2, 3, 4], dtype=np.int16)))
```

Ruff (0.11.9)

Undefined name `np`

(F821)